app/gui/controller/double-representation/src/node.rs

```diff
@@ -147,6 +148,28 @@ pub fn locate_many<'a>(
     }
 }
 
+// ===================
+// === NodeAstInfo ===
+// ===================
```

Comment:
Three blank lines before section.

Reply:
Damn, I wonder why `enso-formatting` is not fixing it for me...

Reply:
Enso formatter is fixing only top of the files. It stops fixing it as soon as imports block end. We never implemented more.